packages/babel-plugin-transform-inline-environment-variables/__tests__/inline-env-var-test.js

```diff
@@ -21,6 +20,32 @@ describe("inline-env-plugin", () => {
   `
   );
 
+  thePlugin(
+    "should inline environment variables in the whitelist when provided",
+    `
+    process.env.NODE_ENV
+  `,
+    `
+    "development";
+  `,
+    {
+      plugins: [[envInliner, { whitelist: ["NODE_ENV"] }]]
+    }
+  );
+
+  thePlugin(
+    "should not inline environment variables if not present in the whitelist when provided",
+    `
+    process.env.NODE_ENV
+  `,
+    `
```

Comment:
but there is a third argument, should I replace the second by `null`, `undefined` or something else ?